templates/certification/model-details.html

```diff
@@ -62,52 +62,91 @@ <h1>{{ vendor }} {{ name }}</h1>
   </div>
   <div class="row">
     <div class="col-12">
-      <h2>Releases</h2>
+      <h2>Release{% if release_details["releases"]|length > 1 %}s{% endif %}</h2>
       <aside class="p-accordion">
         <ul class="p-accordion__list">
           {% for release in release_details["releases"] %}
-          <li class="p-accordion__group">
-            <div role="heading" aria-level="3" class="p-accordion__heading">
-              <button type="button" class="p-accordion__tab" style="background-position: top 1.25rem left 1rem;"id="tab{{ loop.index }}" aria-controls="tab{{ loop.index }}-section" aria-expanded="{% if release_details['releases']|length == 1 %}true{% else %}false{% endif %}"><h3 class="u-no-margin--bottom">{{ release.name }}</h3></button>
-            </div>
-            <section class="p-accordion__panel" id="tab{{ loop.index }}-section" aria-hidden="{% if release_details['releases']|length == 1 %}false{% else %}true{% endif %}" aria-labelledby="tab{{ loop.index }}">
-              {% if release.level == "Enabled" %}
-                <p class="u-no-margin--bottom">Pre-installed in some regions with a custom Ubuntu image that takes advantage of the system’s hardware features and may include additional software. Standard images of Ubuntu may not work well, or at all.</p>
-              {% elif release.level == "Certified" %}
-                <p>The {{ vendor }} {{ name }} development board with the components described below has been awarded the status of certified for Ubuntu.</p>
-                <p class="u-no-margin--bottom">
-                  <a href="{{ release.download_url }}" class="p-button--neutral">Download</a>
+            {% if release_details["releases"]|length == 1 %}
+            <h3>{{ release.name }}</h3>
+            {% if release.level == "Enabled" %}
+            <p class="u-no-margin--bottom">Pre-installed in some regions with a custom Ubuntu image that takes advantage of the system’s hardware features and may include additional software. Standard images of Ubuntu may not work well, or at all.</p>
+            {% elif release.level == "Certified" %}
+            <p>The {{ vendor }} {{ name }} development board with the components described below has been awarded the status of certified for Ubuntu.</p>
+            <p class="u-no-margin--bottom">
```

Comment:
Paragraphs inside of `<ul>`?

Reply:
And you are still using the accordion classes, so I presume that maybe just removing `p-accordiong__group` will work? and if keep the same structure, maybe we don't need to duplicate HTML

Reply:
I've refactored to conditionally render the accordion classes, thanks!